### ISSUE TITLE
[🚫 Bug] findSetting(): cannot set value if setting is out of focus

### ISSUE THREAD

**Issue by ronsig** [redhat-developer/vscode-extension-tester#1424]
### Describe the bug

[Related to issue #1243]
I called 
`let setting = await WorkBench.openSettings().findSetting('Auto Save', 'Files') as ComboSetting;`
The returned setting is valid, but if the screen reslution/size cause it to be off-screen, I can't modify it using 
`await setting.setValue(state); `
I receive an error: ElementNotInteractableError: element not interactable.
If I manually bring it into focus while debugging - setValue() succeeds

I tried sending "Page Down", "Tab", "Down" keys to the driver - but nothing works I'm able to send a click() command to the setting, That sets focus to it, but it's still not fully in view, and I can't send it "setValue()" or sendKeys() commands.

### Steps to reproduce

```Shell
Call `let setting = await WorkBench.openSettings().findSetting('Auto Save', 'Files') as ComboSetting;`
Call `await setting.setValue(state); `

=> An error is thrown
```

### Logs

```Shell
None
```

### Operating System

Windows 11

### Visual Studio Code

1.91.1

### vscode-extension-tester

8.0.2

### NodeJS

18.13

**Comment by djelinek** [comment]
Hello, thank you for reporting that.. could you please share the screenshot how much is setting of the screen? It would help me to reproduce locally.

Thank you

**Comment by ronsig** [comment]
Sure.  As you can see in the screenshot, the test opens the auto save setting, but I cannot modify 'Auto Save' since it is not visible.

```
let stateSetting = await workBench.getDriver().wait(async () => {
            return await settingsEditor.findSetting('Auto Save', 'Files') as ComboSetting;
        }, 10000); // keep trying for 10 seconds
```

> Hello, thank you for reporting that.. could you please share the screenshot how much is setting of the screen? It would help me to reproduce locally.
> 
> Thank you

![image](https://github.com/user-attachments/assets/b122f625-b8bb-43f5-b902-67a78aad3569)

**Comment by djelinek** [comment]
@ronsig thank you for the screenshot 🙂 

I can see the main issue is that searched setting is not displayed as first, because there is an exact match so `File: Auto Save` should be at the top.

I was able to reproduced locally, so I will try to provide a fix.

**Comment by djelinek** [comment]
The fix was shipped in v8.4.0

@ronsig Could you please verify and let me know if it is working better for you?

Thanks
